src/rgw/setup/rgw.py

```diff
@@ -90,7 +90,7 @@ def prepare(conf: MappedConf):
         return 0
 
     @staticmethod
-    def config(conf: MappedConf):
+    def config(conf: MappedConf, index: str = '1'):
```

Comment:
why do we need this index value?

Reply:
If we use the const.MOTR_ADMIN_FID_KEY value to create addb dir then, I think we can remove this index parameter.

Reply:
agreed. I removed index parameter.

Reply:
agreed. I removed index parameter.